test/Dockerfile.pylint

```diff
@@ -0,0 +1,16 @@
+FROM metabrainz/python:3.5
+
+RUN mkdir /data
+RUN mkdir /code
+WORKDIR /code
+
+# Python dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+                       build-essential \
+                       git
+COPY . /code/
+RUN pip install -r requirements.txt
+RUN pip install -r requirements_dev.txt
+
+CMD find . -iname "*.py" | xargs pylint -j 4 | tee /data/pylint.out
```

Comment:
Same idea as with regular tests. No volumes, just copy the report directly out of the container and feed it to the Jenkins plugin.